### PR TITLE
CI tests with Actions

### PR DESCRIPTION
**Pull request by matteobachetti**

### CODE REVIEW

**Comment by matteobachetti** [comment]
Tests are not starting, I guess there is something in the settings that blocks Actions from running?

**Comment by paulray** [comment]
Interesting, I didn't know switching to actions was so easy.  Maybe @scottransom or Joe Glaser know what setting to change?

**Comment by aarchiba** [comment]
The Github actions documentation all seems to suggest it should be picked up automatically and appear under the Actions tab, but it's not showing up, as far as I can tell, on either the main PINT repository or matteobachetti/PINT.

**Comment by matteobachetti** [comment]
@paulray could you check if Actions are disabled in Settings ->Actions in the repo, or in Organization settings > Member privileges?

(See https://github.community/t/how-can-i-disable-a-github-action/17049)

**Comment by matteobachetti** [comment]
Tests are now running in my repo: https://github.com/matteobachetti/PINT/runs/1575808988?check_suite_focus=true

**Comment by bshapiroalbert** [comment]
I just helped switch over the pulsar signal simulator from Travis CI to GitHub Actions yesterday (https://github.com/PsrSigSim/PsrSigSim/pull/182) and ran into this same issue. From the documentation here https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull-request-events-for-forked-repositories , basically the "trigger" to start the tests happens only in the repo where the PR is going to, so if the `.github/workflows/test.yml` is in a forked repo, but not in the main repo, or the one target repo of the PR, they won't run. To get them to run I had to merge the PR that adds the GitHub Actions CI test without any tests running, and then to check that they run and pass submit a new PR where I basically just added a comment to some file (https://github.com/PsrSigSim/PsrSigSim/pull/183). 

tl;dr: The GitHub Actions CI tests won't run unless they exist in both the source and destination repo.

**Comment by paulray** [comment]
I'm happy to merge this if it will help

**Comment by bshapiroalbert** [comment]
> I'm happy to merge this if it will help

If @matteobachetti is happy with the tests that are there and they are passing to an acceptable degree then I would say merge them. Then I'd pull that update, and submit a short PR (maybe just a comment addition) to check that they actually do run, and see what's passing and what needs to be added to the tests, if anything. Not sure if merging this will make these tests run on other currently open PRs without pulling this update though. I will say there is basically no wait time and they ran way faster than the Travis tests are currently running for the signal simulator at least, so it's definitely worthwhile to do.

**Comment by paulray** [comment]
Let's merge then. We can always revert if needed. But, I'm ready to be done with Travis.

**Comment by matteobachetti** [comment]
Ooops sorry I didn't see this. Sure, I'm very happy about the merge!
